src/richie-front/js/utils/filters/isResourceBasedFilterGroupName.ts

```diff
@@ -0,0 +1,14 @@
+import { includes } from 'lodash-es';
```

Comment:
Alas, importing from lodash like this gets us the whole kitchen sink.

We can do `import includes from 'lodash-es/includes'` to only import the relevant function. It does not look as good but since importing the whole lodash kitchen sink would increase payload size by 50+% I think it's a reasonable tradeoff.

(That's AFAIK the best way. If you know a way to have clean imports and not get the kitchen sink please share it 🙌)

Reply:
Oh yes you are right, I forgot that we can import only the code wanted and all the lodash code base. :+1: